docs/index.md

```diff
@@ -5,8 +5,9 @@ nav_order: 1
 ---
 # Orkestra
 
-Find it on [Github](https://github.com/Azure/orkestra)
+ Fork us on [Github](https://github.com/Azure/orkestra)
 
+[![Slack](https://img.shields.io/badge/slack-slack-brightgreen)](https://join.slack.com/t/azureorkestra/shared_invite/zt-rowzrite-Hm_eaih4GyjjZXWftuoqPQ)
```

Comment:
Is there a better way to add this so it isn't just a button? I feel like buttons are traditionally for code-related things and this is for community engagement

Reply:
Do we have good examples of how other projects do this @helayoty?

Reply:
I suppose this might be a better way - https://github.com/openservicemesh/osm#community